src/enveloping/NonceQueueSelector.ts

```diff
@@ -0,0 +1,66 @@
+import { FeesTable } from './FeeEstimator'
```

Comment:
i think this class has too much hard-coded stuff, which can maybe be improved for a later iteration.

i also think it depends too much on the index, i think it'd be better for it to depend on something related to the worker objects themselves